### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by wslyvh**
Adding more resources. Mainly useWeb3 - a curated overview of web3 and solidity development 
This repo is listed on there as well :) 

## Checklist

- [ ] The URL is not already present in the list (check with CTRL/CMD+F in the raw markdown file).
- [ ] Each description starts with an uppercase character and ends with a period.<br>Example: `solc-js - JavaScript bindings for the compiler.`
- [ ] Drop all `A` / `An` prefixes at the start of the description.
- [ ] Avoid using the word `Solidity` in the description.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -142,6 +144,7 @@
 - [paulrberg/solidity-template](https://github.com/paulrberg/solidity-template) - Github template for writing contracts (uses: Hardhat, TypeChain, Ethers, Waffle, Solhint, Solcover, Prettier Plugin Solidity).
 - [tomhirst/solidity-nextjs-starter](https://github.com/tomhirst/solidity-nextjs-starter) - A full-stack dApp starter built with Next.js (React).
 - [ZumZoom/solidity-template](https://github.com/ZumZoom/solidity-template) - Hardhat template with preconfigured Github Actions and Coveralls support.
+- [Typescript Solidity Dev Starter Kit](https://github.com/rhlsthrm/typescript-solidity-dev-starter-kit) - This is a starter kit for developing, testing, and deploying smart contracts with a full Typescript environment.
```
Comment (bkrem) [inline README.md:147]:
```suggestion
- [rhlsthrm/typescript-solidity-dev-starter-kit](https://github.com/rhlsthrm/typescript-solidity-dev-starter-kit) - Starter kit for developing, testing, and deploying smart contracts with a full Typescript environment. 
```

File: README.md [hunk 2]
```diff
@@ -81,6 +81,7 @@
 - [Syntax cheat sheet](https://topmonks.github.io/solidity_quick_ref/) - Quick syntax overview.
 - [Solidity and Vyper cheat sheet](https://reference.auditless.com/cheatsheet) - Review syntax of both languages side-by-side.
 - [willitscale/learning-solidity](https://github.com/willitscale/learning-solidity) - Complete guide on getting started, creating your own crypto, ICOs and deployment.
+- [useweb3 tutorials](https://www.useweb3.xyz/tutorials) - A curated list of free, community tutorials that are based around specific projects, tasks or challenges.
```
Comment (bkrem) [inline README.md:84]:
```suggestion
- [useweb3.xyz/tutorials](https://www.useweb3.xyz/tutorials) - A curated list of free, community tutorials that are based around specific projects, tasks or challenges.
```

**Review by bkrem — CHANGES_REQUESTED** [review]
Hey @wslyvh, love what you've built with `useWeb3`! Thank you for including the Awesome Solidity list there too :)

Two minor suggestion diffs around how the links themselves are presented, then good to go 🚀

**Comment by wslyvh** [comment]
Cool. Update both suggestions.

And thanks for the kind words. Appreciate it :)

**Review by bkrem — APPROVED** [review]

**Pull request merged**